app.js

```diff
@@ -0,0 +1,146 @@
+var express = require('express');
+var app = express();
+var bodyParser = require('body-parser');
+
+
+var server = app.listen(3000, () => {
+  console.log('Server listening.');
+});
+
+
+class BuzzWordArray {
+  constructor (){
+    this.buzzWords =[]
+  }
+
+  find(word){
+    var found = false;
+    this.buzzWords.forEach((wordObj, index, array) => {
+      if (wordObj.buzzWord === word){
+        console.log('found a match:', wordObj.buzzWord, word);
+        console.log('find returning the wordObj', wordObj);
+        found = wordObj;
+      }
+    });
+
+    return found;
+  }
+
+  delete(buzzWord){
+    console.log('running buzzword.delete()');
+    console.log('parameter is ', buzzWord);
+    let found = false;
+    this.buzzWords.forEach((wordObj, index, array) => {
+      if (wordObj.buzzWord === buzzWord){
+        console.log('found a match:', wordObj.buzzWord, buzzWord);
+        array.splice(index, 1);
+        found = true;
+      }
+    });
+    return found;
+  }
+
+  reset(){
+    this.buzzWords = [];
+  }
+}
+
+
+var buzzWords = new BuzzWordArray();
+
+
+var player1 = {
+  score: 0
+}
+
+
+class BuzzWord {
+  constructor (buzzWord, points){
+    this.buzzWord = buzzWord;
+    this.points = points;
+    this.heard = 0;
+  }
+
+  save() {
+    buzzWords.buzzWords.push(this);
+  }
+
+  incrementHeard(){
+    this.heard++;
+  }
+}
+
+
+app.use(express.static('public'));
+
+
+app.use(bodyParser.urlencoded({ extended: false }));
+
+
+app.use(bodyParser.json());
```

Comment:
If this is not used it should be removed, otherwise all requests are being passed through it.